custom-connectors/PixelnebulaFlare/apiDefinition.swagger.json

```diff
@@ -0,0 +1,215 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Flare Digital Signage",
+    "description": "Digital signage software for professional results, in moments. Connect to Flare to manage your digital screens.",
+    "version": "1.0",
+    "contact": {
+      "name": "Flare Support",
+      "url": "https://pixelnebula.com/contact",
+      "email": "support@pixelnebula.com"
+    }
+  },
+  "host": "spark-dev.pixelnebula.com",
+  "basePath": "/",
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://pixelnebula.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://pixelnebula.com/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Communication;Marketing"
+    }
+  ],
+  "schemes": [
+    "http"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/takeovers": {
+      "get": {
+        "description": "Gets a list of active takeovers on your Flare account.",
+        "responses": {
+          "200": {
+            "description": "OK response",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "lifetime": {
+                    "type": "object",
+                    "properties": {
+                      "start": {
+                        "type": "string",
+                        "description": "The start of the takeover lifetime."
+                      },
+                      "end": {
+                        "type": "string",
+                        "description": "The end of the takeover lifetime."
+                      }
+                    },
+                    "description": "lifetime"
+                  },
+                  "_id": {
+                    "type": "string",
+                    "description": "Unique takeover ID."
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "Name of the takeover."
+                  }
+                }
+              }
+            }
+          },
+          "default": {

```

Comment:
@chrishatherall We would recommend you to replace **ALL** the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
@sriyen-msft The certification submission docs say "_must not have default responses with an exact schema definition_". Does this mean no default responses at all? I'm happy to remove the defaults entirely if that's recommended.

Reply:
@chrishatherall You can have a default response, it is just that the default response should not have an exact schema definition. Other responses, such as `200`, `403`, etc., can have an exact schema definition. You should define those expected responses so that when a response comes in, it's more likely that they will match against those expected definitions, rather than the default response.

Reply:
@dougelliott I have these default entries with an empty schema. If this is incorrect, what should the non-exact schema definition look like? I'm having trouble finding information about this.

Reply:
@chrishatherall What you currently have is fine as far as the default responses :)

What Srikanth is suggesting is that you also add the expected status codes for _all_ the codes you would reasonably expect in normal operation, with a defined schema of the responses. So in the end, this would leave you with default responses with no exact schema, and expected responses (`200`, `403`, etc.) with an exact schema. You already have defined the responses for `200`, so you could simply expand on that for other responses that might come through.